scrape/scrape.go

```diff
@@ -355,12 +397,20 @@ func (sp *scrapePool) reload(cfg *config.ScrapeConfig) error {
 	sp.client = client
 
 	targetScrapePoolTargetLimit.WithLabelValues(sp.config.JobName).Set(float64(sp.config.TargetLimit))
+	targetScrapePoolLabelLimit.WithLabelValues(sp.config.JobName).Set(float64(sp.config.TargetLimit))
```

Comment:
I would not expose this as metric because we do not expose the total number of labels.

Reply:
good point, I removed them